src/features/goblins/tailor/TabContent.tsx

```diff
@@ -64,12 +64,12 @@ export const TabContent: React.FC<Props> = ({ tab }) => {
   const items = tab === "collection" ? collection : upcoming;
 
   const goToUpcomingDrops = () => {
-    window.open("https://testnet.bumpkins.io/#/upcoming-drops", "_blank");
+    window.open("https://bumpkins.io/#/upcoming-drops", "_blank");
```

Comment:
Can you make that if the .env network is  "mumbai", it redirects to the testnet website, and if it points to "mainnet" redirect to normal bumpkins website.